cmd/iam.go

```diff
@@ -179,7 +179,7 @@ func (sys *IAMSys) initStore(objAPI ObjectLayer, etcdClient *etcd.Client) {
 
 	if etcdClient == nil {
 		var group *singleflight.Group
-		if env.Get("_MINIO_IAM_SINGLE_FLIGHT", config.EnableOff) == config.EnableOn {
+		if env.Get("_MINIO_IAM_SINGLE_FLIGHT", config.EnableOn) == config.EnableOn {
```

Comment:
So singleflight is on by default now.

Reply:
yes, I am moving it back since that is not the real cause of any problems @donatello